pkg/merge/openapi.go

```diff
@@ -88,6 +89,11 @@ func walkerMergeSpec(dst *OpenAPISpec) filepath.WalkFunc {
 			return nil
 		}
 
+		if ext := strings.ToLower(filepath.Ext(path)); ext != ".yaml" && ext != ".yml" {
+			// skip because this is not a YAML file
+			return nil
```

Comment:
Let's add a debug log for these skips, what do you think?

Reply:
done